docs/content/en/_index.md

```diff
@@ -25,23 +25,127 @@ cascade:
         weight: 20
 ---
 
-<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable no-inline-html -->
 <!-- markdownlint-disable-next-line line-length -->
-{{< blocks/cover title="Welcome to the Keptn Lifecycle Toolkit Documentation" image_anchor="top" height="half" color="primary" >}}
+{{< blocks/cover title="" image_anchor="top" height="full" color="primary" >}}
 <div class="mx-auto">
- <a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
+<div class="mb-4 d-none d-md-block " >
+<picture >
+    <img alt="keptn" src="/images/home/hero/keptn-logo-white.svg">
+</picture>
+</div>
+  <h1 class="display-1 mt-0 mt-md-5 pb-4">Lifecycle Toolkit</h1>
+  <h2 class="styled display-3 mt-0 mb-4">Cloud native application lifecycle orchestration </h2>
+<div>
+ <a class="btn btn-lg -bg-green mr-3 mb-4" href="{{< relref "/docs" >}}">
   Docs <i class="fas fa-arrow-alt-circle-right ml-2"></i>
  </a>
-    <a class="btn btn-lg btn-primary mr-3 mb-4" href="https://github.com/keptn/lifecycle-toolkit/releases">
+    <a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/keptn/lifecycle-toolkit/releases">
   Releases <i class="fab fa-github ml-2 "></i>
  </a>
 </div>
+</div>
+<div class="usecasebox">
+
+## Use Cases
+
+We extend the K8s APIs with the following Use Cases:
+
+<div class="row usecases">
+{{% blocks/feature icon="home homeobservability" title="Deployment Observability" %}}
+make ANY Kubernetes Deployment OBSERVABLE
+
+<!-- markdownlint-disable-next-line link-fragments -->
+ <a class="btn btn-lg -bg-light mr-3 mb-4" href="#deployment-observability">
+  read more <i class="fas fa-arrow-alt-circle-down ml-2"></i>
+ </a>
+{{% /blocks/feature %}}
+{{% blocks/feature icon="home homedata" title="Deployment Data Access" %}}
+standardize access for all Observability Data for K8s
+
+<!-- markdownlint-disable-next-line link-fragments -->
+ <a class="btn btn-lg -bg-light mr-3 mb-4" href="#data-access">
+  read more <i class="fas fa-arrow-alt-circle-down ml-2"></i>
+ </a>
+{{% /blocks/feature %}}
+
+{{% blocks/feature icon="home homeorchestrate" title="Orchestrate Deployment Checks" %}}
+orchestrate deployment checks as part of scheduler
```

Comment:
I don't like much this sentence, but I don't have also a better phrase :/

Reply:
Maybe something like "Gain confidence in your work with pre-/post-deployment checks"
Don't know if this fits size wise
CC: @aepfli

Reply:
![image](https://user-images.githubusercontent.com/9987394/234530200-06592e90-7a33-4972-bce5-ac453e74da18.png)